F-FrontEnd/src/F-output-xcodeml.c

```diff
@@ -1058,16 +1058,23 @@ static void outx_lineno(lineno_info *li)
 /**
  * output tag with format and lineno attribute
  */
+static void outx_vtagLineno_null(int l, const char *tag, lineno_info *li)
+{
+    static char buf1[CHAR_BUF_SIZE], buf2[CHAR_BUF_SIZE];
+    snprintf(buf1, sizeof(buf1), "<%s", tag);
+    strncpy(buf2, buf1, sizeof(buf2));
+    
+    outx_indent(l);
+    outx_puts(buf2);
+    outx_lineno(li);
+}
+
 static void outx_vtagLineno(int l, const char *tag, lineno_info *li,
                             va_list args)
```

Comment:
I think it's better to define `outx_vtagLineno` as follows than add `outx_vtagLineno_null`.
```suggestion
static void outx_vtagLineno(int l, const char *tag, lineno_info *li, ...)
```